pkgs/by-name/ew/eww/package.nix

```diff
@@ -32,18 +47,34 @@ rustPlatform.buildRustPackage rec {
     librsvg
   ];
 
-  cargoBuildFlags = [ "--bin" "eww" ];
+  cargoBuildFlags = [
+    "--bin"
+    "eww"
+  ];
 
   cargoTestFlags = cargoBuildFlags;
 
   # requires unstable rust features
   RUSTC_BOOTSTRAP = 1;
 
   meta = {
-    description = "ElKowars wacky widgets";
+    description = "A widget system made in Rust to create widgets for any WM";
+    longDescription = ''
+      Eww (ElKowar's Wacky Widgets) is a widget system made in Rust, which lets
+      you create your own widgets similarly to how you can in AwesomeWM.
+      The key difference: It is independent of your window manager!
+      It can be configured in yuck and themed using CSS, very easy
```

Comment:
I'd change this to "... using CSS, **is** very easy ..."

Reply:
Done.